jsonapi/unmarshal.go

```diff
@@ -320,6 +312,30 @@ func UnmarshalInto(input map[string]interface{}, targetStructType reflect.Type,
 	return nil
 }
 
+// check if there is any field tag with the given name available
+func getFieldByTagName(val reflect.Value, fieldName string) (field reflect.Value, found bool) {
+	for x := 0; x < val.NumField(); x++ {
+		// check if there is an embedded struct which needs to be searched
+		if val.Field(x).CanInterface() && val.Field(x).CanAddr() {
+			_, isEmbedded := val.Field(x).Addr().Interface().(UnmarshalIdentifier)
```

Comment:
you are probably right, i will just change this, we only need to know if it `CanAddr()` and `CanAddr().CanInterface()`